sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -1549,6 +1561,1188 @@
+                            "refreshOn": "",
+                            "dataGridLabel": false,
+                            "widget": null,
+                            "validateOn": "change",
+                            "showCharCount": false,
+                            "showWordCount": false,
+                            "allowMultipleMasks": false,
+                            "addons": [],
+                            "inputType": "radio",
+                            "fieldSet": false,
+                            "id": "edde5i9",
+                            "defaultValue": "",
+                            "lockKey": true
+                        },
+                        {
+                            "title": "Other exceptional circumstance Panel",
+                            "theme": "default",
+                            "breadcrumb": "default",
+                            "breadcrumbClickable": false,
+                            "buttonSettings": {
+                                "previous": false,
+                                "cancel": false,
+                                "next": false
+                            },
+                            "scrollToTop": false,
+                            "tooltip": "",
+                            "customClass": "",
+                            "collapsible": false,
+                            "hidden": false,
+                            "hideLabel": true,
+                            "disabled": false,
+                            "modalEdit": false,
+                            "key": "otherExceptionalPartnerCircumstancePanel",
+                            "tags": [],
+                            "properties": {},
+                            "customConditional": "",
+                            "conditional": {
+                                "json": "",
+                                "show": "true",
+                                "when": "reasonsignificantdecreaseInPartnerIncome",
+                                "eq": "other"
+                            },
+                            "nextPage": "",
+                            "logic": [],
+                            "attributes": {},
+                            "overlay": {
+                                "style": "",
+                                "page": "",
+                                "left": "",
+                                "top": "",
+                                "width": "",
+                                "height": ""
+                            },
+                            "type": "panel",
+                            "label": "Other exceptional circumstance Panel",
+                            "tabindex": "",
+                            "input": false,
+                            "tableView": false,
+                            "components": [
+                                {
+                                    "label": "Please explain your situation:",
```

Comment:
May be 
![image](https://github.com/user-attachments/assets/a53445b3-6f22-4698-9832-b7c3b908e1e1)